creator-node/test/sync.test.js

```diff
@@ -354,9 +355,7 @@ describe('Test secondarySyncFromPrimary()', async function () {
     })
 
     describe('Confirm export works for user with data exceeding maxExportClockValueRange', async function () {
-      /**
-       * override maxExportClockValueRange to smaller value for testing
-       */
+      /** Override maxExportClockValueRange to smaller value for testing */
       beforeEach(async function () {
         maxExportClockValueRange = 10
         process.env.maxExportClockValueRange = maxExportClockValueRange
```

Comment:
this is probably the only test that uses that value but I'm cautious about this impacting other tests if we don't reset it in an afterEach. like if sync test expects it to be something else by default but this test gets run before it then it would be a pain to debug

Reply:
i think we do clean this up? but added one more line in case thats what you meant - lmk

Reply:
ah ok I see where it deletes it. wasn't familiar with this way for tests since I usually set it in the config and then mock and re-set the config to the original value. your way removes the need for mocking tho so that's good. I think this should be fine